client/db/bolt/upgrades.go

```diff
@@ -366,6 +369,68 @@ func v6Upgrade(dbtx *bbolt.Tx) error {
 	})
 }
 
+// v7Upgrade sets the status to all archived refund matches to order.MatchConfirmed.
+// From now on all refunds must also have confirmations for the trade to be
+// considered inactive.
+func v7Upgrade(dbtx *bbolt.Tx) error {
+	const oldVersion = 6
+
+	if err := ensureVersion(dbtx, oldVersion); err != nil {
+		return err
+	}
+
+	amb := []byte("matches") // archived matches
+	mKey := []byte("match")  // matchKey
+	pKey := []byte("proof")  // proofKey
```

Comment:
yes. For the upgrade I would rather freeze the exact values in time. It would be weird if the values the variables pointed to would change, I agree, but just to be safe.